src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -506,7 +535,7 @@ public String getPublicationDate() {
             return null;
         }
 
-        String year = getField("year");
+        String year = YearUtil.toFourDigitYear(getField("year"));
```

Comment:
Do not update data value without notifying the user. If you want to propose such a change, please open a new PR.